### PR TITLE
Initialise project

### PR DESCRIPTION
**Pull request by nabeelamir-defra**
https://eaflood.atlassian.net/browse/IWTF-2699

Setup rod-catch-returns-api-js project

### CODE REVIEW

**Review by lailien3 — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+6. Issue that pull request!
+
+> When raising a pull request, ensure that you apply the appropriate label as this is used when generating changelogs!
+
+## Any contributions you make will be under the Open Government Licence
+
+In short, when you submit code changes, your submissions are understood to be under the same
+[OGLv3 License](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3) that covers the project.
+
+## Report bugs using Github's [issues](https://github.com/DEFRA/rod-licensing/issues)
```
Comment (irisfaraway) [inline CONTRIBUTING.md:25]: This still links to the rod-licencing repo

**Review by irisfaraway — CHANGES_REQUESTED** [review]
One small thing!

File: CONTRIBUTING.md (hunk 1 above)
Comment (nabeelamir-defra) [reply CONTRIBUTING.md]: Totally missed that, this is now fixed.

**Review by jaucourt — APPROVED** [review]

**Pull request merged**